src/python/dxpy/cli/parsers.py

```diff
@@ -226,23 +226,85 @@ class PrintInstanceTypeHelp(argparse.Action):
+                  '                                         "*":    "mem1_ssd1_v2_x4"}\'')
+            print()
+            print(fill('2. Runs all entry points of the first stage with ' +
+                       'mem2_hdd2_v2_x2, the main entry point of the second stage with mem1_ssd1_v2_x4, ' +
+                       'the stage named "BWA" with mem1_ssd1_v2_x2, and all other ' +
                        'stages with mem2_hdd2_v2_x4'))
             print()
-            print('    Example: dx run workflow --instance-type 0=mem2_hdd2_v2_x2 \\')
-            print('               --instance-type BWA=mem1_ssd1_v2_x2 --instance-type mem2_hdd2_v2_x4')
+            print('    dx run workflow-xxxx \\\n' +
+                  '     --instance-type 0=mem2_hdd2_v2_x2 \\\n' +
+                  '     --instance-type 1=\'{"main": "mem1_ssd1_v2_x4"}\' \\\n' +
+                  '     --instance-type BWA=mem1_ssd1_v2_x2 \\\n' +
+                  '     --instance-type mem2_hdd2_v2_x4')
+            print()
+            print(fill('--instance-type-by-executable argument is a JSON string with a double mapping that ' +
+                       'specifies instance types by app or applet id, then by entry point within the executable.' +
+                       'This specification applies across the entire nested execution tree and is propagated ' +
+                       'across /executable-xxxx/run calls issued with the execution tree.'))
+            print()
+            print('More examples')
+            print(fill('3. Force every job in the execution tree to use mem2_ssd1_v2_x2'))
+            print()
+            print(
+                '    dx run workflow-xxxx --instance-type-by-executable \'{"*": {"*": "mem2_ssd1_v2_x2"}}\'')
+            print()
+            print(fill(
+                '4. Force every job in the execution tree executing applet-xyz1 to use mem2_ssd1_v2_x2'))
+            print()
+            print(
+                '    dx run workflow-xxxx --instance-type-by-executable \'{"applet-xyz1":{"*": "mem2_ssd1_v2_x2"}}\'')
+            print()
+            print(fill('5. Force every job executing applet-xyz1 to use mem2_ssd1_v2_x4 ' +
+                       'for the main entry point and mem2_ssd1_v2_x2 for all other entry points.' +
+                       'Also force the collect entry point of all executables other than applet-xyz1 to use mem2_ssd1_v2_x8.' +
+                       'Other entry points of executable other than applet-xyz1 may be overridden by ' +
+                       'lower-priority mechanisms'))
+            print()
+            print('    dx run workflow-xxxx --instance-type-by-executable \\\n' +
+                  '           \'{"applet-xyz1":  {"main":    "mem2_ssd1_v2_x4", "*": "mem2_ssd1_v2_x2"},\n' +
+                  '             "*":            {"collect": "mem2_ssd1_v2_x8"}}\'')
+            print()
+            print(fill('6. Force every job executing applet-xxxx to use mem2_ssd1_v2_x2 for all entry points' +
```

Comment:
Please add a space at the end of the line to avoid `pointsin`.

Reply:
Fixed. Thanks for catching it!